bridge/README.md

```diff
@@ -140,3 +144,49 @@ black .
 ```
 docker build .
 ```
+
+## Mlflow Quickstart
+
+This is a quick guide to running Mlflow locally for testing. A full Mlflow
+installation consists of 3 components:
+
+* Mlflow tracking server / model registry
+* Database backend.
+* Storage backend. 
+
+While it is possible to run just the Mlflow server, the database backend is
```

Comment:
Considering this is targeted at people without MLFlow, I think this is too dense. You could just say:

Bridge requires all three components. The instructions below will initialize the server and database locally and use an S3 bucket for artifact storage.

(I.E, avoid temptation to explain all options, at this point we want people to get to something that works asap)

Reply:
Idk, this was something that took me 30 minutes to figure out from Mlflow docs, and I think it would be pretty valuable to a new reader. If they don't understand it they can just skip to the steps below.